Translations/pt_BR.po

```diff
@@ -1364,6 +1357,7 @@ msgstr "Reportar erros em https://github.com/diasurgical/devilutionX/"
 #: Source/diablo.cpp:1103
 msgid "Please update devilutionx.mpq and fonts.mpq to the latest version"
 msgstr ""
+"Por favor, atualize devilutionx.mpq e fonts.mpq para a versão mais recente"
```

Comment:
IF we want to be consistent with the one below, I'd also include the articles here:
"Por favor, atualize o devilutionx.mpq e o fonts.mpq para a versão mais recente"